### PR TITLE
Arpin uncomment delete

### PR DESCRIPTION
**Pull request by djarpin**
We've decided to uncomment out endpoint deletion as a best practice for examples.

### CODE REVIEW

File: sagemaker-python-sdk/mxnet_gluon_sentiment/sentiment_analysis_with_mxnet_gluon.ipynb [hunk 1]
```diff
@@ -209,7 +209,7 @@
    },
    "outputs": [],
    "source": [
-    "# m.delete_endpoint()"
+    "m.delete_endpoint()"
```
Comment (winstonaws) [inline sagemaker-python-sdk/mxnet_gluon_sentiment/sentiment_analysis_with_mxnet_gluon.ipynb]: I don't think we want to keep this method on the estimator longer term. Can you change this to use the session to delete it?

File: introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-fulltraining.ipynb [hunk 2]
```diff
@@ -566,8 +566,8 @@
    },
    "outputs": [],
    "source": [
-    "#sage.delete_endpoint(EndpointName=endpoint_name)\n",
-    "#sage.delete_model(ModelName=model_name)"
+    "sage.delete_endpoint(EndpointName=endpoint_name)\n",
+    "sage.delete_model(ModelName=model_name)"
```
[the remark this answers is not in the document]
Comment (djarpin) [reply introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-fulltraining.ipynb]: Updated.

File: introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-transfer-learning.ipynb [hunk 3]
```diff
@@ -581,8 +581,8 @@
    },
    "outputs": [],
    "source": [
-    "#sage.delete_endpoint(EndpointName=endpoint_name)\n",
-    "#sage.delete_model(ModelName=model_name)"
+    "sage.delete_endpoint(EndpointName=endpoint_name)\n",
+    "sage.delete_model(ModelName=model_name)"
```
[the remark this answers is not in the document]
Comment (djarpin) [reply introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-transfer-learning.ipynb]: Updated.

File: sagemaker-python-sdk/mxnet_gluon_sentiment/sentiment_analysis_with_mxnet_gluon.ipynb (hunk 1 above)
Comment (djarpin) [reply sagemaker-python-sdk/mxnet_gluon_sentiment/sentiment_analysis_with_mxnet_gluon.ipynb]: Updated.

**Pull request merged**